### PR TITLE
feat(CS1172) - searchbox + hits example

### PR DESCRIPTION
**Pull request by pedr-clinia**

### CODE REVIEW

File: examples/default/src/components/OpeningHours.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      } else if (now.isBefore(startHour)) {
+        openingHoursText = (
+          <>
+            <span className="open">Open today: </span>
+            <span>
+              {startHour.format('LT')} - {endHour.format('LT')}
+            </span>
+          </>
+        );
+      } else {
```
Comment (FelixLeChat) [inline examples/default/src/components/OpeningHours.js]: You should check if it's closed for now right ? now is after endhour

**Comment by etiennecl** [comment]
C’est principaement le boilerplate de create-react-app!

File: examples/default/src/components/AutoSuggest.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  }
+
+  onChange = (_, { newValue }) => {
+    this.setState({
+      value: newValue,
+    });
+  };
+
+  onSuggestionsFetchRequested = async ({ value }) => {
+    const results = await this.props.client.search(value, {
```
Comment (capitainecl) [inline examples/default/src/components/AutoSuggest.js]: I think this should be client.suggest

Comment (etiennecl) [reply examples/default/src/components/AutoSuggest.js]: Pas dans la version qu’il a actuellement

File: examples/default/src/App.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+const ExampleHitComponent = ({ searchResult }) => {
+  const {
+    type,
+    address: { streetAddress, place, regionCode },
+    geoPoint,
+    phones,
+    openingHours,
+  } = searchResult;
+
+  const mapLink = `https://www.google.ca/maps/dir/${streetAddress.replace(
```
Comment (FelixLeChat) [inline examples/default/src/App.js]: This is not the right link.

`https://www.google.ca/maps/dir//289+Boulevard+Desaulniers,+Saint-Lambert,+QC+J4P+1M8/`

there are two // after dir and the lat and lng are the position of the user (inserted by google map). You don't have to specify those

**Comment by capitainecl** [comment]
@FelixLeChat Can you rereview this PR so we can close it? :)

**Pull request merged**